### ISSUE TITLE
Support the Escape key for cancelling changes

### ISSUE THREAD

**Issue by asoltesz** [Ilogeek/redmine_issue_dynamic_edit#75]
It would be nice to have the editing dialogs closed on pressing the Escape key (as usually expected with dialogs)

**Comment by Ilogeek** [comment]
Hi @asoltesz 

I'm glad my plugin gives you so much ideas and we can definitely add some to improve the plugin!
Don't hesitate to submit your PR if you already have implemented some of them and I'll merge your PR into the project as soon as possible.
I'll take a look for this specific feature, escape key press is something interesting to add to the plugin 

If you find the plugin helpful, take a minute to star it on Github and rate it 5 stars on http://www.redmine.org/plugins/redmine_issue_dynamic_edit 👍

Have a good day too

**Comment by asoltesz** [comment]
@Ilogeek Yes, dynamic_edit is a great productivity booster makes one enthusiastic. (thanks for the plugin BTW)

I would love to contribute, although I have nonexistent Ruby/Rails skills and limited in Javascript. I will take a look and see if I can implement this.
